test/benchee/statistics_test.exs

```diff
@@ -57,6 +67,8 @@ defmodule Benchee.StatistcsTest do
       stats
     end
 
+    # no mode asserts in the sample asserts as the samples have no duplicated
+    # values and therefore mode isn't a useful value
```

Comment:
Yup I read those as well, but I'm not sure if this will actually provide more value to people. Imo in that case we can just return one or the other and we don't have to really care about it - I don't think it makes that big of a difference for the end user.

for the case when there's no mode... true. Maybe nil, could be the best - true.

Hm, I might add the detections of this to the code, I worry a bit though about the additional execution with not much knowledge gained for the end user (both the nil value or the multiple modes don't provide that much value to the end user imo)